src/main/java/com/arpnetworking/tsdcore/sinks/DimensionFilteringSink.java

```diff
@@ -33,6 +34,12 @@
      */
     @Override
     public void recordAggregateData(final PeriodicData data) {
+        if (_whitelistDimensions.isPresent()) {
+            if (!_whitelistDimensions.get().containsAll(data.getDimensions().keySet())) {
```

Comment:
Hm, welp, whitelist usually mean 'allow things on the list, disallow anything not on the list" - which is what we want to do here. My intention here is to allow metrics that have only a partial set of dimensions on the whitelist - i.e. if a metric has `service` and `cluster` but no `host`, then let it through, as I would expect happens today. I'm not sure this is the place to be enforcing that metrics contain at least `service`, `cluster` and `host` - shouldn't that happen in MAD? or on the entry to point to CAgg?